bot/tasks/pennychat.py

```diff
@@ -197,11 +279,13 @@ def _penny_chat_details_blocks(penny_chat_invitation, mode=None):
     if include_calendar_link:
         start_date = penny_chat_invitation.date.astimezone(utc).strftime('%Y%m%dT%H%M%SZ')
         end_date = (penny_chat_invitation.date.astimezone(utc) + timedelta(hours=1)).strftime('%Y%m%dT%H%M%SZ')
+
+        description = f'{penny_chat_invitation.description}\nVideo Link: {penny_chat_invitation.video_conference_link}'
         google_cal_url = 'https://calendar.google.com/calendar/render?' \
                          'action=TEMPLATE&text=' \
-            f'{urllib.parse.quote(penny_chat_invitation.title)}&dates=' \
-            f'{start_date}/{end_date}&details=' \
-            f'{urllib.parse.quote(penny_chat_invitation.description)}'
+                         f'{urllib.parse.quote(penny_chat_invitation.title)}&dates=' \
+                         f'{start_date}/{end_date}&details=' \
+                         f'{urllib.parse.quote(description)}'
```

Comment:
this function has grown in complexity, can you pull out the contents of this if statement into it's own helpful func that lives in this file?